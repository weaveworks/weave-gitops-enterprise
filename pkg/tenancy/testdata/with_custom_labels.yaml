tenants:
  - name: foo-tenant
    namespaces:
    - foo-ns
    - bar-ns
    labels:
      example.com/test-label: testing
