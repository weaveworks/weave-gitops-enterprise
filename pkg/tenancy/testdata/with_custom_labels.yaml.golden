apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: foo-ns
spec: {}
status: {}
---
apiVersion: v1
kind: ServiceAccount
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: foo-tenant
  namespace: foo-ns
---
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: foo-tenant
  namespace: foo-ns
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: User
  name: gotk:foo-ns:reconciler
- kind: ServiceAccount
  name: foo-tenant
  namespace: foo-ns
---
apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: bar-ns
spec: {}
status: {}
---
apiVersion: v1
kind: ServiceAccount
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: foo-tenant
  namespace: bar-ns
---
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  creationTimestamp: null
  labels:
    example.com/test-label: testing
    toolkit.fluxcd.io/tenant: foo-tenant
  name: foo-tenant
  namespace: bar-ns
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: User
  name: gotk:bar-ns:reconciler
- kind: ServiceAccount
  name: foo-tenant
  namespace: bar-ns
---
